docs/version0.9.md

```diff
@@ -22,12 +22,108 @@
+
+
+## New GC policy
+
+A new GC policy is introduced for [JEP 318: Epsilon: A No-Op Garbage Collector](http://openjdk.java.net/jeps/318).
+
+When this policy is enabled, the Java object heap is expanded in the normal way until the limit is
+reached, but memory is not reclaimed through garbage collection. When the limit is reached the VM shuts down.
+
+This JEP has a number of use cases including short-lived applications and certain test scenarios.
+
+To enable this policy you can use one of the following options:
+
+- [-Xgcpolicy:nogc](#xgcpolicy.md)
+- [-XX:+UseNoGC]()
+
+## Modifying the default Java heap size for applications that run in containers
+
+When using container technology, applications are typically run on their own and do not need to compete for memory. In this release, changes
+are introduced to detect when OpenJ9 is running inside a container. If your application is running in a container and
+you want the VM to allocate a larger fraction of memory to the Java heap, set the `-XX:+UseContainerSupport` option on the command line.
+
+The following table shows the maximum Java heap size that gets set, depending on the memory available:
+
+
+| Physical memory `<size>`   | Maximum Java heap size (v0.9.0) |
+|----------------------------|---------------------------------|
+| Less than 1 GB             | 50% `<size>`                    |
+| 1 GB - 2 GB                | `<size>` - 512M                 |
+| Greater than 2 GB          | 75% `<size>`                    |
+
+
+## Specifying the maximum Java heap size as a percentage value
+
+OpenJ9 now supports setting the heap size as a percentage of the physical memory. The following OpenJDK options are recognized and can be
+set for the VM:
+
+- ![Start of content that applies to Java 8 and later](cr/java8plus.png) [-XX:MaxRAMPercentage](add link when topic available) ![End of content that applies to Java 8 and later](cr/java_close_lts.png)
+- ![Start of content that applies only to Java 9 and later](cr/java9plus.png) [-XX:InitialRAMPercentage](add link when topic available) ![End of content that applies only to Java 9 or later](cr/java_close.png)
+
+## Shared classes support for nested jar files
+
+Changes are made to the `com.ibm.oti.shared` API to support nested jar files.
+
+## Direct Dump Reader enabled
+
+Direct Dump Reader (DDR) support is now enabled for the OpenJ9 VM on all platforms. The DDR code enables the VM to read system dump data by using the OpenJ9 Diagnostic Tool
+Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool. If you use the [Eclipse Memory Analyzer Tool (MAT)](https://www.eclipse.org/mat/), you can also analyze OpenJ9 or IBM VMs by installing the DTFJ plugin.
+(Install from the Eclipse Help menu; Install New Software > Work with "IBM Diagnostic Tool Framework for Java" > IBM Monitoring and  Diagnostic Tools > Diagnostic Tool Framework for Java)
+
+## Changes to the `java.lang.String` class
```

Comment:
We should also mention the big changes to StringBuffer and StringBuilder. We no longer use OpenJ9 implementations of these classes (for Java 9+) and instead pick them up from OpenJDK. This means more compatibility w.r.t. the behavior of these classes.

Reply:
Completed.